clang/lib/Basic/Targets.cpp

```diff
@@ -11,6 +11,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "clang/Config/config.h"
```

Comment:
unneeded include

Reply:
This is needed for the change in SystemZ.h to replace the hard coded `"z10"` string with the CLANG_SYSTEMZ_DEFAULT_ARCH cmake variable.  This applies to the other comment too.

Reply:
If SystemZ.h uses CLANG_SYSTEMZ_DEFAULT_ARCH, that file should have `#include "clang/Config/config.h"` as IWYU

Reply:
That was my preference too except clang/Config/config.h has a clause that it can only be included once and will generate an error if it is included a second time.